src/aat/java/uk/gov/hmcts/ccd/datastore/befta/DataStoreTestAutomationAdapter.java

```diff
@@ -74,6 +74,21 @@ public Object calculateCustomValue(BackEndFunctionalTestScenarioContext scenario
                     .computeIfAbsent(scenarioContext.getContextId(), k ->
                     UUID.randomUUID().toString());
         }
+        else if (key.toString().startsWith("approximately ")) {
+                try {
+                    String actualValue = (String) ReflectionUtils.deepGetFieldInObject(scenarioContext,
+                        "testData.actualResponse.body.__plainTextValue__");
+                    String expectedValue = key.toString().replace("approximately ", "");
+                    int actualSize = actualValue.length();
+                    int expectedSize = Integer.parseInt(expectedValue);
+                    if (Math.abs(actualSize-expectedSize)<50)
```

Comment:
How about checking difference against a percentage of the actual size?
`Math.abs(actualSize-expectedSize) < actualSize * 10 / 100`